### ISSUE TITLE
Uncaught TypeError: Cannot set property 'value' of null  

### ISSUE THREAD

**Issue by robertsLando** [spiritree/vue-orgchart#14]
I get this error when I click on an element.

```
Uncaught TypeError: Cannot set property 'value' of null
    at ft (vue-orgchart.min.js?f5c3:1)
    at HTMLDivElement.eval (vue-orgchart.min.js?f5c3:1)
```

Seems the problem is that 

```
document.getElementById("selected-node");
```

Always returns null

**Comment by spiritree** [comment]
you should add this in your <template>
```
<div v-show="false">
    <label class="selected-node-group">选中树节点</label>
    <input type="text" id="selected-node" class="selected-node-group">
 </div>
```

**Comment by robertsLando** [comment]
@spiritree I'm still not able to update the org-chart. When I change chart-data it is not updated, how can I trigger the update?

**Comment by spiritree** [comment]
Please describe more or give a demo.

**Comment by robertsLando** [comment]
```
<template>
      <div>
        <vo-edit
        :data="chartData"
        :direction="direction"
        :chartClass="chartClass"
        :nodeContent="nodeContent"
        >
      </vo-edit>

      <!--Selected node container-->
      <div v-show="false">
        <label class="selected-node-group"></label>
        <input type="text" id="selected-node" class="selected-node-group">
      </div>
    </div>
  </template>
```

When I update `chartData` the chart doesn't reload with new data provided

**Comment by spiritree** [comment]
@robertsLando 
#10 Do you mean that?

**Comment by robertsLando** [comment]
@spiritree Like you can see in my example the only way to trigger an update is to hide/show the org chart. Yes my problem is like the one of #10 but I was looking for a better solution, I think the component itself shoul watch for data changes and update itself

**Comment by spiritree** [comment]
I have fixed the problem in #10 
19cc16bfe96f4ab29c5d28ac9c478b7f24dc6f2e
